src/components/organisms/ContentBox.jsx

```diff
@@ -157,8 +157,8 @@ const FullscreenContainer = styled.div`
   position: absolute;
   top: 0;
   left: 0;
-  height: 100vh;
```

Comment:
Careful here. Did you check all the different content types (Text, Media, Gigs) to show correctly on different screen sizes?